File: problems/c/chanukah/chanukah.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    p = int(input())
    for _ in range(p):
        k, n = [int(val) for val in input().split()]

        candles = [candle + 1 for candle in range(1, n + 1)]
        print(k, sum(candles))


if __name__ == '__main__':
    main()

File: problems/c/chanukah/chanukah.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int p, k, n, candles;

    cin >> p;

    for (int i=0; i<p; i++) {
        cin >> k;
        cin >> n;

        candles = 0;
        for (int j=1; j<n+1; j++) {
            candles += j + 1;
        }

        cout << k << " " << candles << endl;

    }

    
}